### PR TITLE
created testSetFixed validator

### PR DESCRIPTION
**Pull request by hailzrene**

### CODE REVIEW

File: src/test/java/edu/kcc/animal/AnimalTest.java [hunk 1]
```diff
@@ -166,7 +169,11 @@ public void testGetFixed(){
     // TODO
     @Test
     public void testSetFixed() {
-        fail("The test case is a prototype.");
+        System.out.println("setFixed");
+        boolean fixed = false;
+        Animal instance = new Animal();
+        instance.setFixed(fixed);
```
Comment (mlhaus) [inline src/test/java/edu/kcc/animal/AnimalTest.java:175]: The default value for fixed is false. Why are you trying to set false to false? Set false to true.

**Review by mlhaus — CHANGES_REQUESTED** [review]
I'd like this test to verify that an unfixed animal can be set to fixed -- false to true.

File: src/test/java/edu/kcc/animal/AnimalTest.java (hunk 1 above)
Comment (hailzrene) [reply src/test/java/edu/kcc/animal/AnimalTest.java:180]: I have fixed it but am getting an error when trying to push it through again, is there a different method to upload my changes?

Comment (mlhaus) [reply src/test/java/edu/kcc/animal/AnimalTest.java:180]: Try adding and commiting your changes first. Then pull all of the changes from my repo with `git pull upstream main`. Then do a push to your repo.

**Comment by hailzrene** [comment]
This has the fixed version of last weeks pull as well as this weeks due to the pom.xml error.

**Pull request merged**